mathics/builtin/assignment.py

```diff
@@ -9,8 +9,10 @@
 from mathics.builtin.base import (
     Builtin, BinaryOperator, PostfixOperator, PrefixOperator)
 from mathics.core.expression import (Expression, Symbol, valid_context_name,
-                                     system_symbols)
-from mathics.core.rules import Rule
+                                     system_symbols,String)
+
```

Comment:
Can you fix the formatting here. Should be: `system_symbols, String`

Reply:
Done